community/modules/file-system/gke-persistent-volume/main.tf

```diff
@@ -20,12 +20,16 @@ locals {
 }
 
 locals {
-  location             = split("/", var.filestore_id)[3]
-  filestore_name       = split("/", var.filestore_id)[5]
+  is_gcs = (var.gcs_bucket_name != null)
+
+  filestore_id         = !local.is_gcs ? var.filestore_id : "projects/empty/locations/empty/instances/empty"
```

Comment:
Is there a special meaning to the `"empty"` values?

Reply:
It is arbitrary. The logic of the module is that it authors both pv and pvc for GCS and Filestore. Then at the end it only applies the one that it is actually using. If this value is null then all subsequent calculations become ugly as we check for null on each one. I added a comment in the code to clarify that the value is arbitrary.